packages/analytics-browser/src/browser-client.ts

```diff
@@ -27,7 +27,7 @@ import {
 import { convertProxyObjectToRealObject, isInstanceProxy } from './utils/snippet-helper';
 import { Context } from './plugins/context';
 import { useBrowserConfig, createTransport } from './config';
-import { webAttributionPlugin } from '@amplitude/plugin-web-attribution-browser';
+import { WebAttribution } from '@amplitude/analytics-client-common';
```

Comment:
Since web attribution is web only, maybe it should be in `analytics-browser` package?

Reply:
Then in web-attribution plugin, we need to install analytics-browser? I don't think that's efficient.

Reply:
I saw we have campaign-tracker in the analytics-client-common which has similar logic as web-attribution. So it should not only for the web, it's also used in react-native-client.
I think it can be in '@amplitude/analytics-client-common' but should have a better name.
Maybe campaign-tracker-v2?

Reply:
> web-attribution plugin, we need to install analytics-browser

I don't think that's too much of an issue since `plugin-web-attribution-browser` only works with `analytics-browser`, it will always have that dependency anyway.

Not a huge deal either way, but since this only works for browser it could help remove confusion by moving it out of core

Reply:
Sync offline. 
React native also has web, which makes more sense to keep the logic in `analytics-client-common`.
Since React native doesn't have 2.0.0 yet, we can wait the changes until we have react-native to 2.0.